src/RequestAPI.d.ts

```diff
@@ -51,6 +51,12 @@ export type RequestAPI = {
 
   "settings::write": <K extends keyof Settings>(key: K, value: any) => void;
   "settings::get": <K extends keyof Settings>(key: K) => Optional<any>;
+  "settings::getos": () => NodeJS.Platform;
```

Comment:
This is not really a setting, so maybe this instead?
```suggestion

  "os::platform": () => NodeJS.Platform;
```